src/deployers/PluginsDeployerFactory.cpp

```diff
@@ -29,51 +29,51 @@ PluginsDeployerFactory::PluginsDeployerFactory(AppDir& appDir,
                                                                       qtTranslationsPath(std::move(qtTranslationsPath)),
                                                                       qtDataPath(std::move(qtDataPath)) {}
 
-std::shared_ptr<PluginsDeployer> PluginsDeployerFactory::getInstance(const std::string& moduleName) {
+std::vector<std::shared_ptr<PluginsDeployer>> PluginsDeployerFactory::getDeployers(const std::string& moduleName) {
```

Comment:
A `set` might've worked as well. But it doesn't really matter here, as the deployers can't be compared anyway right now.